src/lua/api-gateway/tracking/VariableManager.lua

```diff
@@ -50,14 +50,8 @@ function _M:getResponseVariable(response_var, cache)
         return cache[response_var]
     end
 
-    local ngx_header = ngx.header[response_var]
-    if ngx_header ~= nil then
```

Comment:
Why is this capability removed ? IIRC this was allowing backend services to manage throttling policies with variables created by the service, so that the Gateway doesn’t have to look in the response.

Reply:
This has been removed due to the fact that the first upstream throttling capability was supposed to act only on the upstream response `status` and not other response data. It will be bundled in future releases though, but we want to minimise the code impact.

Reply:
I see. If the plan is to add it back it’s ok.